truenas/webui
# NAS-112108 / 21.10 / Check VM support on page loading

Prerequisites:
- Use recent SCALE build

How to test:

- Go to the Virtualization page
- If your system has virtualization support you'll see no changes
- Otherwise, you'll see entity empty with `Virtualization is not supported` title and no `Add` actions
